lib/start.js

```diff
@@ -108,8 +108,18 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
 
   if (options.network_log) {
     let exitCode = 0
+    let jsonOutput = {}
     // Read the network log
-    const jsonOutput = fs.readJsonSync(networkLogFile)
+    const jsonContent = fs.readFileSync(networkLogFile, 'utf8')
+    try {
+      jsonOutput = JSON.parse(jsonContent)
+    } catch (e) {
+      // fix the json if there was a parsing error
+      const n = jsonContent.lastIndexOf('},')
+      const fixedJSONContent = jsonContent.substring(0, n) + "}]}"
+      jsonOutput = JSON.parse(fixedJSONContent)
```

Comment:
This also can be within a `try | catch` block, but haven't seen it fail till now.

Reply:
This doesn't seem to guarantee the JSON file is completely written by the network log process, just that it is parseable. What if instead it tried to re-parse the file some number of times with a time delay to wait until the JSON file is completely rewritten?

Reply:
I tried `reparsing` the file 3 times after a wait of 10 secs on `Win10`, but I always got the file with terminating string `},`. I'll try increasing the timeout to check if it may be a result of my VM being slow.

Reply:
@jumde i see, so it seems the file is indeed complete but just that it has a weird terminating string on windows?

Reply:
in that case i suggest adding a comment in the code to say that this is a windows chromium issue